pkg/codegen/docs/gen.go

```diff
@@ -914,7 +914,7 @@ func (mod *modContext) genConstructorPython(r *schema.Resource, argsOptional, ar
 			Type: propertyType{
 				Name:            typeName,
 				DescriptionName: descriptionName,
-				Link:            "#inputs",
+				Link:            "#inputs/",
```

Comment:
I'm not sure we need trailing slashes on the anchor links, because I don't think those get trailing slashes added by Hugo

Reply:
@kimberleyamackenzie to make sure i understand. so google does not consider www.website.com/#inputs, and www.website.com/#inputs/ as different things?